cmd/dm-ctl/main.go

```diff
@@ -31,24 +31,35 @@ import (
 	"github.com/pingcap/dm/pkg/utils"
 )
 
+func helpUsage(cfg *common.Config) {
+	fmt.Println("Usage: dmctl [global options] command [command options] [arguments...]")
+	fmt.Println()
+	ctl.PrintUsage()
+	fmt.Println()
+	fmt.Println("Special Commands:")
+	f := cfg.FlagSet.Lookup(common.EncryptCmdName)
+	fmt.Println(fmt.Sprintf("\t--%s %s", f.Name, f.Usage))
+	fmt.Println()
+	fmt.Println("Global Options:")
+	cfg.FlagSet.VisitAll(func(flag2 *flag.Flag) {
+		if flag2.Name == common.EncryptCmdName {
+			return
+		}
+		fmt.Println(fmt.Sprintf("\t--%s %s", flag2.Name, flag2.Usage))
+	})
+}
+
 func main() {
 	cfg := common.NewConfig()
-	err := cfg.Parse(os.Args[1:])
-	switch errors.Cause(err) {
-	case nil:
-	case flag.ErrHelp:
-		os.Exit(0)
-	default:
-		fmt.Printf("parse cmd flags err: %s", err)
-		os.Exit(2)
-	}
-
-	utils.PrintInfo2("dmctl")
-	fmt.Println() // print a separater
+	args := os.Args[1:]
 
+	syncErr := log.L().Sync()
```

Comment:
why sync log here?

Reply:
I found it has been removed already, maybe this PR takes a long time to review